### PR TITLE
Feature: minor enhancement including updating fabric binaries setup script to use v2.2.0

### PR DESCRIPTION
**Pull request by ThomasRalee**
### This PR:
- Updated instruction on the new UI enhancement (service tabs).
- Added queryApprove command to list of chaincode.sh.
- Added peer chaincode invoke transaction beside fabric-sdk invoke transaction example.
- Added metadata to contract for more descriptive inputs on auto generated forms.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,35 +32,27 @@ From the project root directory, download the Hyperledger Fabric peer binaries b
 Note that downloading the binaries will take some time.
 
 In the `webapp/server/.env` file, replace the `ORDERER_NAME`, `PEER_NAME`, and `MSP_ID` variables with:
-
-* `ORDERER_NAME` — the orderer name of the network you deployed your node in. To access the orderer name—in the [Chainstack UI](https://console.chainstack.com/), navigate to your Hyperledger Fabric network, click **Details**, copy the **Orderer name** value.
-* `PEER_NAME` — the peer name of the node that you deployed. To access the peer name—in the [Chainstack UI](https://console.chainstack.com/), navigate to your Hyperledger Fabric node, copy the **Peer name** value.
-* `MSP_ID` — the Membership Service Provider identity (MSP ID) of the node that you deployed. To access the MSP ID—in the [Chainstack UI](https://console.chainstack.com/), navigate to your Hyperledger Fabric node, copy the **MSP ID** value.
+* `ORDERER_NAME` — the orderer name of the network you deployed your peer in. To access the orderer name—in the [Chainstack UI](https://console.chainstack.com/), navigate to Hyperledger Fabric **service nodes** tab from the network, access **Orderer**, copy the **Orderer name** value.
```
Comment (d2phap) [inline README.md:35]:
```suggestion
* `ORDERER_NAME` — the orderer name of the network you deployed your peer in. To access the orderer name, in the [Chainstack UI](https://console.chainstack.com/), from Hyperledger Fabric network, click on *Service nodes** tab and **Orderer** to access details page. Here you can copy the **Orderer name** value.
```

File: README.md [hunk 2]
```diff
@@ -32,35 +32,27 @@ From the project root directory, download the Hyperledger Fabric peer binaries b
...
+* `PEER_NAME` — the peer name of the peer that you deployed. To access the peer name—in the [Chainstack UI](https://console.chainstack.com/), navigate to the Hyperledger Fabric **Peer nodes** tab from the network, access your peer and copy the **Peer name** value.
```
Comment (d2phap) [inline README.md:36]:
```suggestion
* `PEER_NAME` — the name of the peer that you deployed. To access the peer name, in the [Chainstack UI](https://console.chainstack.com/), from Hyperledger Fabric network, select **Peer nodes** tab, click on your peer name to access its details page. Here you can copy the **Peer name** value.
```

File: README.md [hunk 3]
```diff
@@ -32,35 +32,27 @@ From the project root directory, download the Hyperledger Fabric peer binaries b
...
+* `MSP_ID` — the Membership Service Provider identity (MSP ID) of the peer that you deployed. To access the MSP ID—in the [Chainstack UI](https://console.chainstack.com/), navigate to the Hyperledger Fabric **Peer nodes** tab from the network, access your peer and copy the **MSP ID** value.
```
Comment (d2phap) [inline README.md:37]:
```suggestion
* `MSP_ID` — the Membership Service Provider identity (MSP ID) of the peer that you deployed. To access the MSP ID, in the [Chainstack UI](https://console.chainstack.com/), from Hyperledger Fabric network, select **Peer nodes** tab, click on your peer name to access its details page. Here you can copy the **MSP ID** value.
```

File: README.md [hunk 4]
```diff
@@ -32,35 +32,27 @@ From the project root directory, download the Hyperledger Fabric peer binaries b
...
+1. Export the [required files](./webapp/certs/README.md) from [Chainstack UI](https://console.chainstack.com/).
 
-Set up the web app client:
+1. Set up the web app client:
 
 ```sh
 cd webapp/client
 npm i
 npm run build
 ```
```
Comment (d2phap) [inline README.md:53]: Add 2 spaces for indentation

File: webapp/certs/README.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+### Export the required files:
+
+On the [Chainstack UI](https://console.chainstack.com/):
```
Comment (d2phap) [inline webapp/certs/README.md:3]:
```suggestion
In [Chainstack Console](https://console.chainstack.com/):
```

File: webapp/certs/README.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+1. Network connection profile:
+  -  Navigate to your Hyperledger Fabric network.
+  - click **Details**.
+  - Click **Export connection profile**.
```
Comment (d2phap) [inline webapp/certs/README.md:8]: 4 spaces

```suggestion
    - Navigate to your Hyperledger Fabric network.
    - Click **Details**.
    - Click **Export connection profile**.
```

File: webapp/client/src/components/ActionButton.vue [hunk 7]
```diff
@@ -84,3 +85,9 @@ export default {
   },
 };
 </script>
+
+<style lang="scss" scoped>
+  .v-btn {
+    font-family: Roboto,sans-serif!important;
```
Comment (d2phap) [inline webapp/client/src/components/ActionButton.vue:91]:
```suggestion
    font-family: Roboto, sans-serif !important;
```

**Review by d2phap — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -32,41 +32,33 @@ From the project root directory, download the Hyperledger Fabric peer binaries b
...
 
-In the `webapp` directory, create a `certs` directory.
-
-In the [Chainstack UI](https://console.chainstack.com/):
-
-* Navigate to your Hyperledger Fabric network, click **Details**. Click **Export orderer TLS certificate**. Click **Export connection profile**.
-* Navigate to your Hyperledger Fabric node. Next to **Organization identity**, click **Export**. Unzip the downloaded file.
-
-Copy the three exported files to the `webapp/certs/` directory you created previously.
+1. Export the [required files](./webapp/certs/README.md) from [Chainstack UI](https://console.chainstack.com/).
```
Comment (akegaviar) [inline README.md:45]: Why move the export instructions to a separate file?

File: webapp/certs/README.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
+### Export the required files:
+
+In [Chainstack Console](https://console.chainstack.com/):
```
Comment (akegaviar) [inline webapp/certs/README.md:3]: Please change to `Chainstack UI` for consistency.

File: README.md (hunk 8 above)
Comment (ThomasRalee) [reply README.md:45]: I created a readme file in the certs folder so the participants don't have to create a certs folder.
Hence i thought it made more sense to move the instructions there.

**Review by d2phap — APPROVED** [review]

File: README.md [hunk 10]
```diff
@@ -32,41 +32,33 @@ From the project root directory, download the Hyperledger Fabric peer binaries b
...
 
 In the `webapp/server/.env` file, replace the `ORDERER_NAME`, `PEER_NAME`, and `MSP_ID` variables with:
-
-* `ORDERER_NAME` — the orderer name of the network you deployed your node in. To access the orderer name—in the [Chainstack UI](https://console.chainstack.com/), navigate to your Hyperledger Fabric network, click **Details**, copy the **Orderer name** value.
-* `PEER_NAME` — the peer name of the node that you deployed. To access the peer name—in the [Chainstack UI](https://console.chainstack.com/), navigate to your Hyperledger Fabric node, copy the **Peer name** value.
-* `MSP_ID` — the Membership Service Provider identity (MSP ID) of the node that you deployed. To access the MSP ID—in the [Chainstack UI](https://console.chainstack.com/), navigate to your Hyperledger Fabric node, copy the **MSP ID** value.
+* `ORDERER_NAME` — the orderer name of the network you deployed your peer in. To access the orderer name, in the [Chainstack Platform](https://console.chainstack.com/), from Hyperledger Fabric network, select **Service nodes** tab and click on **Orderer** to access its details page. Here you can copy the **Orderer name** value.
```
Comment (easeev) [inline README.md:35]: There's no "Chainstack Platform", there's either "Chainstack" or "Chainstack platform"

File: README.md [hunk 11]
```diff
@@ -32,41 +32,33 @@ From the project root directory, download the Hyperledger Fabric peer binaries b
...
 
-In the `webapp` directory, create a `certs` directory.
-
-In the [Chainstack UI](https://console.chainstack.com/):
-
-* Navigate to your Hyperledger Fabric network, click **Details**. Click **Export orderer TLS certificate**. Click **Export connection profile**.
-* Navigate to your Hyperledger Fabric node. Next to **Organization identity**, click **Export**. Unzip the downloaded file.
-
-Copy the three exported files to the `webapp/certs/` directory you created previously.
+1. Export the [required files](./webapp/certs/README.md) from [Chainstack Platform](https://console.chainstack.com/).
```
Comment (easeev) [inline README.md:45]: Same as above

File: webapp/certs/README.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
+### Export the required files:
+
+In the [Chainstack Platform](https://console.chainstack.com/):
```
Comment (easeev) [inline webapp/certs/README.md:3]: Same as above

File: README.md [hunk 13]
```diff
@@ -32,41 +32,33 @@ From the project root directory, download the Hyperledger Fabric peer binaries b
...
+* `ORDERER_NAME` — the orderer name of the network you deployed your peer in. To access the orderer name, in the [Chainstack Platform](https://console.chainstack.com/), from Hyperledger Fabric network, select **Service nodes** tab and click on **Orderer** to access its details page. Here you can copy the **Orderer name** value.
+* `PEER_NAME` — the name of the peer that you deployed. To access the peer name, in the [Chainstack Platform](https://console.chainstack.com/), from Hyperledger Fabric network, select **Peer nodes** tab, click on your peer name to access its details page. Here you can copy the **Peer name** value.
+* `MSP_ID` — the Membership Service Provider identity (MSP ID) of the peer that you deployed. To access the MSP ID, in the [Chainstack Platform](https://console.chainstack.com/), from Hyperledger Fabric network, select **Peer nodes** tab, click on your peer name to access its details page. Here you can copy the **MSP ID** value.
```
Comment (easeev) [inline README.md:37]: Same as above

**Pull request merged**